Xamarin.Essentials/MediaPicker/MediaPicker.android.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Android.App;
+using Android.Content;
+using Android.Content.PM;
+using Android.Provider;
+
+namespace Xamarin.Essentials
+{
+    public static partial class MediaPicker
+    {
+        const int requestCodeMediaPicker = 12348;
+        const int requestCodeMediaCapture = 12349;
```

Comment:
At some point we need to make sure we don't have any clashes of codes in our library.

Reply:
Yeah I actually had the thought of moving this into a simple Platform internal API where we can just go `NextRequestCode()` and it manages them.  Maybe should do that quick.